services/ui/src/components/SshKeys/AddSshKey.js

```diff
@@ -80,7 +83,7 @@ const AddSshKey = ({me: { id, email }}) => {
                   type="text"
                   onChange={handleChange}
                   value={values.sshKey}
-                  placeholder="Begins with 'ssh-rsa', 'ssh-ed25519'"/>
+                  placeholder="Begins with 'ssh-rsa', 'ssh-ed25519', 'ecdsa-sha2-nistp256', 'ecdsa-sha2-nistp384', 'ecdsa-sha2-nistp521'"/>
```

Comment:
Did you happen to run the UI to look at this change? It's a really long placeholder now and I'm not sure how well it scales with length.

Reply:
That text field is configured to wrap on dashes or spaces, so it should be good for most modern browser window sizes, I'd expect?

![Screen Shot 2022-04-01 at 12 01 09 PM](https://user-images.githubusercontent.com/5650531/161309289-9ff33eaa-9a9c-468c-aa07-373482a6e329.png)

Reply:
Not gonna win any design awards but as long as it's functional...